src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (number) => {
+    if (number === undefined) {
```

Comment:
The parameter `number` should be explicitly checked against `undefined` using the strict equality operator `===` to avoid unintended coercion. You've done this correctly here.